client-src/elements/chromedash-intent-content.js

```diff
@@ -0,0 +1,212 @@
+    this.intentBody = '';
+  }
+
+  static get styles() {
+    return [
+      ...SHARED_STYLES,
+      css`
+        #copy-email-body {
+          cursor: pointer;
+          color: var(--link-color);
+        }
+        .email-content-border {
+          border: 1px solid #ddd;
+          box-shadow: rgba(0, 0, 0, 0.067) 1px 1px 4px;
+          margin: 8px 0 24px 0;
+        }
+        .email-content-div {
+          background: white;
+          padding: 12px;
+        }
+        p {
+          color: #444;
+        }
+        h3 {
+          margin-bottom: 10px;
+          &:before {
+            counter-increment: h3;
+            content: counter(h3) '.';
+            margin-right: 5px;
+          }
+        }
+        #content section > div {
+          background: white;
+          border: 1px solid #ddd;
+          box-shadow: rgba(0, 0, 0, 0.067) 1px 1px 4px;
+          padding: 12px;
+          margin: 8px 0 16px 0;
+        }
+        #content section > p {
+          color: #444;
+        }
+
+        .email .help {
+          font-style: italic;
+          color: #aaa;
+        }
+        .email h4 {
+          font-weight: 600;
+        }
+        .alertbox {
+          margin: 2em;
+          padding: 1em;
+          background: var(--warning-background);
+          color: var(--warning-color);
+        }
+        .subject {
+          font-size: 16px;
+        }
+        table {
+          tr[hidden] {
+            th,
+            td {
+              padding: 0;
+            }
+          }
+
+          th {
+            padding: 12px 10px 5px 0;
+            vertical-align: top;
+          }
+
+          td {
+            padding: 6px 10px;
+            vertical-align: top;
+          }
+
+          td:first-of-type {
+            width: 60%;
+          }
+
+          .helptext {
+            display: block;
+            font-size: small;
+            max-width: 40em;
+            margin-top: 2px;
+          }
+
+          input[type='text'],
+          input[type='url'],
+          input[type='email'],
+          textarea {
+            width: 100%;
+            font: var(--form-element-font);
+          }
+
+          select {
+            max-width: 350px;
+          }
+
+          :required {
+            border: 1px solid $chromium-color-dark;
+          }
+
+          .interacted:valid {
+            border: 1px solid green;
+          }
+
+          .interacted:invalid {
+            border: 1px solid $invalid-color;
+          }
+
+          input:not([type='submit']):not([type='search']) {
+            outline: 1px dotted var(--error-border-color);
+            background-color: #ffedf5;
+          }
+        }
+      `,
+    ];
+  }
+
+  setCopyEmailListener() {
+    const copyEmailBodyEl = this.shadowRoot.querySelector('#copy-email-body');
+    const emailBodyEl = this.shadowRoot.querySelector('.email');
+    if (copyEmailBodyEl && emailBodyEl) {
+      copyEmailBodyEl.addEventListener('click', () => {
```

Comment:
Now that this is in lit, can we use `@click=...` in the `render()` method rather than using `addEventListener()`?  

It is OK to do this in a follow-up PR.  Also, this needs to be typescript.